packages/contracts-bedrock/src/L1/ETHLockbox.sol

```diff
@@ -191,10 +192,7 @@ contract ETHLockbox is ProxyAdminOwnedBase, Initializable, ISemver {
     ///         transaction batch, or otherwise the OptimismPortal may not be able to unlock ETH
     ///         from the ETHLockbox on finalized withdrawals.
     /// @param _lockbox The address of the ETH lockbox to migrate liquidity to.
-    function migrateLiquidity(IETHLockbox _lockbox) external {
-        // Check that the sender is the proxy admin owner.
-        if (msg.sender != proxyAdminOwner()) revert ETHLockbox_Unauthorized();
```

Comment:
In the past we've expressed a preference for an explicit check rather than a modifier.

I can imagine that might be a hassle as I think we'd need to declare new errors in each file, but calling it out for discussion.

Reply:
Agreed with not using a modifier here. We could just have the error be `ProxyAdminOwnedBase_Unauthorized()` (or whatever the correct format is) and just make the modifier into a method, either of the same name, or something like `assertOnlyProxyAdmin`

Reply:
Just want to use this as a chance to clarify - what's the value of this over the modifier? I never really understood that choice

Reply:
This is a good threat summarizing why no modifiers unless you have something before and after the `_;` https://x.com/DrakeEvansV1/status/1708271596207579341

Reply:
Mostly a readability thing for me